features/step_definitions/laundry_room_steps.rb

```diff
@@ -35,4 +35,16 @@
 When(/^I try to access booking link for "([^"]*)"$/) do |date|
   date = Time.parse(date).to_formatted_s(:short)
   visit laundry_room_create_booking_path(@laundry, time: date)
+end
+
+Given("there is a booking from for {string}") do |date|
+  slot = Time.zone.parse(date)
+  slot_to_book = @laundry.schedule
+                     .occurrences(1.week.from_now)
+                     .detect { |occ| occ.beginning_of_hour.localtime == slot.localtime }
+  @user.book! @laundry, time: slot_to_book, amount: 1
+end
+
+Then(/^I should not see the text "([^"]*)"$/) do |text|
+  page.should have_no_content(text)
```

Comment:
Hmm.. Please restrain from too much cut and paste. This assertion is using a syntax that we have not used in the camp and that is a few years old. We use `expect` in our assertions.